src/components/MainSidebar/TimeDimensions.js

```diff
@@ -35,20 +38,13 @@ const TimeDimensions = () => {
     const stageId = useSelector(sGetUiProgramStage)
```

Comment:
Should we rename this here?
```suggestion
    const stageId = useSelector(sGetUiProgramStageId)
```

Reply:
Yes I just figured I'd do that in a separate PR (which I'll do right now)